projects/36/bhxiv/paper.md

```diff
@@ -293,29 +293,29 @@ To overcome this situation, two methods can be developed:
+The BioHackathon provided an appropriate environment to generate outcomes, to discuss current needs and challenges among different stakeholders, and to start talking about collaborations crossing boarders not only within Europe but also to the US, and crossing projects such as SPHN and EJP RD. Importantly, it also provided an opportunity to present and discuss late-breaking results, cutting-edge technologies, on-going projects, and to develop innovative prototypes. The informal setting of the sessions encouraged presenters and participants to engage in discussions about the presented projects. Such discussions provided invaluable inputs for the ongoing and future work of the projects, while offering participants an effective way to broaden their knowledge of the emerging research trends and technologies and to start collaborations with other researchers in academia and industry. This also came with some challenges as the hybrid meeting required communication across the different intercontinental time zones. Our project had 17 participants (10 onsite and 7 online) from different countries and projects/organisations such as the UK, US, NL, SP or CH. We engaged with 10 participants that were new to the project, and 10 that were new to the BioHackathon-EU. We interacted with the #17 BHEU project to discuss GA4GH Beacon API interoperability with GA4GH Phenopackets data standard and the #27 and #30 BHEU projects and with the OWKIN company to discuss FAIR for ML, with the N3C consortium in the US and EGA in EU to provision COVID-19 datasets, with the SPHN project another health data project based on Semantic Web Technologies and FAIR, and we investigated the interoperability of data represented as semantic Phenopackets with the EJP RD project.
 
```

Comment:
Hi @ljgarcia I cannot see what is your change(s) suggestion here. Could you clarify? Thanks

Reply:
projects/organizations --> projects/organisations

Reply:
Thanks!